src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/Tools/Move-AzureRmVhdVM.ps1

```diff
@@ -0,0 +1,237 @@
+<#
+.ExternalHelp AzureRM.Compute.ManagedService-help.xml
+#>
+
+function Move-AzureRmVhdVM
+{
+    param
+    (
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $true)]
+        [string]$HyperVVMName,
+        
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $true)]
+        [string]$ExportPath,
+        
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $true)]
+        [string]$Location,
+        
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $false)]
+        [string]$HyperVServer = 'localhost',
+
+        [Parameter(Mandatory = $false)]
+        [string]$ResourceGroupName = '',
+        
+        [Parameter(Mandatory = $false)]
+        [string]$OSType = 'Windows'
+    )
+
+    $ErrorActionPreference = 'Stop';
+    Try-LoginAzureRmAccount;
```

Comment:
Please simply check the context - you should not log in as part of this cmdlet, but assume a login and print a message if login has not occurred.

Reply:
removed